nomos-services/cryptarchia-consensus/src/lib.rs

```diff
@@ -78,17 +78,19 @@ struct Cryptarchia {
 }
 
 impl Cryptarchia {
-    pub fn new(
-        header_id: HeaderId,
-        ledger_state: LedgerState,
+    /// Initialize a new [`Cryptarchia`] instance.
+    /// [`Cryptarchia`] must always be initialized from genesis.
+    pub fn from_genesis(
+        genesis_id: HeaderId,
+        genesis_ledger_state: LedgerState,
```

Comment:
Is this true? You can still create a new instance from an advance ledger state and the current tip right? I think `new` wasn't wrong here.

Reply:
You certianly can, but at some point we stated that `Cryptarchia` should always be initialised from `genesis` ([Discord](https://discord.com/channels/1111286067413405788/1357607529814691872/1357627429405069383)).

I changed it (back) to `from_genesis` to reinforce the idea that this is the correct use case. You can always make a "wrong" use of it and initialise it to any other arbitrary arguments.

On the other hand, we could revert it to `new` and clarify in the comments that the intended case is to use `genesis`.